src/pages/NewAsset.vue

```diff
@@ -121,39 +122,6 @@ export default {
       const locations = this.locations
       return get(locations, '[0].shortDisplayName', '')
     },
-    step () {
-      const steps = [
-        true, // fictive step 0
-        true,
-        false,
-        false,
-        false,
-      ]
-
-      if (this.name.length >= 1) { // with high debounce to reduce distraction while typing
-        steps[2] = true
-      }
-      if (this.selectedCategory && this.selectedCategory.name && !isNaN(parseInt(this.price))) {
-        steps[3] = true
-      }
-
-      let isValidStartDate = true
-      let isValidEndDate = true
-      if (this.startDate) {
-        isValidStartDate = isValidDateString(this.startDate)
-      }
-      if (this.endDate) {
-        isValidEndDate = isValidDateString(this.endDate)
-      }
-
-      if (!this.showAvailabilityDates || (this.showAvailabilityDates && this.startDate && isValidStartDate && isValidEndDate)) {
-        // endDate is optional, so is quantity
-        steps[4] = true
-      }
-
-      // Index of first falsy step
-      return steps.indexOf(false) >= 0 ? steps.indexOf(false) - 1 : steps.length - 1
-    },
```

Comment:
From what I know, I had to remove the fact that inputs were hidden when they are empty.
In case they just want to keep for the page I will put the code back and use your code for recruiters.